scripts/export-dos-participants.py

```diff
@@ -68,10 +80,13 @@ def inner(record):
 
     pool = ThreadPool(3)
 
+    logger.info(f'Finding User Research Participants suppliers for {FRAMEWORK_SLUG}')
     records = find_all_participants(client, map_impl=pool.imap)
 
+    logger.info(f"Building CSV for {len(records)} User Research Participants suppliers")
     write_csv_with_make_row(
         records,
         make_row(content_manifest),
-        "output/{}-user-research-participants.csv".format(FRAMEWORK_SLUG)
+        os.path.join(OUTPUT_DIR, "user-research-participants-suppliers.csv".format(FRAMEWORK_SLUG)),
```

Comment:
There isn't a site for the format method to insert the framework slug in this string?